app/facades/rev_it_up_facade.rb

```diff
@@ -0,0 +1,27 @@
+class RevItUpFacade
+  attr_reader :data
+
+  def initialize(data)
+    @data = data
+  end
+
+  def name
+    @data[:name]
+  end
+
+  def avg_rating
+    @data[:avg_rating]
+  end
+
+  def address
+    @data[:location][:display_address].join(" ")
+  end
+
+  def price
+    @data[:price]
+  end
+
+  def category
```

Comment:
Added category for the food choice shown on the user's dashboard after the shake it up has happened.